recipe/meta.yaml

```
@@ -23,7 +23,7 @@ requirements:
     - pip
   run:
     - python >={{ python_min }}
-    - gufe >=1.0,<1.4
+    - gufe >=1.0,<1.6
```

I think we are supporting 1.6 in feflow as per https://github.com/OpenFreeEnergy/feflow/actions/runs/17091562569/job/48466265427#step:6:179 

```suggestion
    - gufe >=1.0,<1.7
```